web/package-lock.json

```diff
@@ -11023,17 +11023,16 @@
             }
         },
         "node_modules/puppeteer-core/node_modules/ws": {
-            "version": "8.13.0",
-            "resolved": "https://registry.npmjs.org/ws/-/ws-8.13.0.tgz",
-            "integrity": "sha512-x9vcZYTrFPC7aSIbj7sRCYo7L/Xb8Iy+pW0ng0wt2vCJv7M9HOMy0UoN3rr+IFC7hb7vXoqS+P9ktyLLLhO+LA==",
+            "version": "7.5.10",
```

Comment:
This is... a downgrade here? Is that a problem?

Reply:
Not a problem